extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -478,9 +478,9 @@ describes.sandboxed('amp-ad-network-adsense-impl', {}, () => {
           '^https://googleads\\.g\\.doubleclick\\.net/pagead/ads' +
           '\\?client=ca-adsense&format=[0-9]+x[0-9]+&w=[0-9]+&h=[0-9]+' +
           '&adk=[0-9]+&raru=1&bc=1&pv=1&vis=1&wgl=1' +
-          '(&asnt=[0-9]+-[0-9]+)?(&dff=%22.*?%22)?' +
+          '(&asnt=[0-9]+-[0-9]+)?(&dff=(?:%22.*?%22|\'.*?\'))?' +
           '&prev_fmts=320x50(%2C[0-9]+x[0-9]+)*' +
-          '(&dff=%22.*?%22)?' +
+          '(&dff=(?:%22.*?%22|\'.*?\'))?' +
```

Comment:
You can remove this line entirely. This is only here because of a copy/paste error, I imagine.